dev/devicelab/lib/tasks/dart_plugin_registry_tests.dart

```diff
@@ -70,7 +70,7 @@ class PluginPlatformInterfaceMacOS {
       );
       pluginImplPubspecContent = pluginImplPubspecContent.replaceFirst(
           '    platforms:\n',
-          '    implements: plugin_platform_interface\n'
+          '    implements: plugin_platform_base\n'
```

Comment:
This is a really confusing name. The better fix here would be to change `plugin` to `aplugin` or `someplugin` everywhere in this test, and keep the standard suffixes.